pontoon/base/models.py

```diff
@@ -750,7 +750,8 @@ def for_project_locale(self, project, locale, paths=None, search=None):
                 'marked': entity.marked,
                 'original_plural': entity.string_plural,
                 'marked_plural': entity.marked_plural,
-                'key': entity.key,
+                # Remove source string fom the key
+                'key': entity.key.split('\x04')[0],
```

Comment:
Translate Toolkit (external lib) uses it for joining source file name and source string to form unique string identifier. We're also following the convention for PO files now:
https://github.com/mozilla/pontoon/pull/226/files

Updated comment.